package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "test5repos",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "dependencies": {
```

Comment:
the most important info here is what dependencies your project use.
we will add more later